sync/fromDisk/questions.js

```diff
@@ -36,6 +36,7 @@ function getParamsForQuestion(q) {
         external_grading_entrypoint: (q.externalGradingOptions && q.externalGradingOptions.entrypoint),
         external_grading_timeout: (q.externalGradingOptions && q.externalGradingOptions.timeout),
         external_grading_enable_networking: (q.externalGradingOptions && q.externalGradingOptions.enableNetworking),
+        external_grading_environment: (q.externalGradingOptions && q.externalGradingOptions.environment) || {},
```

Comment:
Nit: we're running Node 14, so we can use optional chaining and nullish coalescing to make this a bit more concise:
```suggestion
        external_grading_environment: q.externalGradingOptions?.environment ?? {},
```